lib/Doctrine/ODM/PHPCR/UnitOfWork.php

```diff
@@ -3777,6 +3786,21 @@ private function generateUuid()
         return $g();
     }
 
+
+    /**
+     * Extracts ReferenceManyCollection type from field metadata
+     *
+     * @param $referenceFieldMetadata
+     * @return string
+     */
+    private function getReferenceManyCollectionTypeFromMetadata($referenceFieldMetadata)
+    {
+        if (isset($referenceFieldMetadata['strategy']) && 'path' === $referenceFieldMetadata['strategy']) {
```

Comment:
i think here we also want to use ReferenceManyCollection::REFERENCE_TYPE_PATH instead of 'path'